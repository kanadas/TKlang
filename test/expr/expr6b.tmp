expr/expr6b.in

Parse              Failed...

syntax error at line 1, column 42 before `['
