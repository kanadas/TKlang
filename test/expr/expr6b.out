expr/expr6b.in
Type mismatch: [int] and [((char*(TV 4.((char*#4)+void)))+void)] in expression:
[2, 3, "lala"]
